pkg/pod/status_test.go

```diff
@@ -238,10 +238,11 @@ func TestSetTaskRunStatusBasedOnStepStatus_sidecar_logs(t *testing.T) {
 			ctx := config.ToContext(context.Background(), &config.Config{
 				FeatureFlags: featureFlags,
 			})
-			var wantErr *multierror.Error
-			wantErr = multierror.Append(wantErr, c.wantErr)
+			wantErr := errors.Join(c.wantErr)
 			merr := setTaskRunStatusBasedOnStepStatus(ctx, logger, []corev1.ContainerStatus{{}}, &c.tr, pod.Status.Phase, kubeclient, ts)
-
+			if merr == nil {
+				t.Fatalf("Expected error but got nil")
+			}
```

Comment:
Since this is no longer a multierror, do you mind renaming the variable to something more accurate? Maybe `gotErr` or `err`?